vendor/bat-native-ledger/src/bat/ledger/internal/database/database_initialize.cc

```diff
@@ -85,6 +86,8 @@ void DatabaseInitialize::ExecuteCreateScript(
     callback(ledger::Result::LEDGER_ERROR);
     return;
   }
+  
+  ledger_->ClearState(ledger::kStateServerPublisherListStamp);
```

Comment:
@NejcZdovc This is ensure that if a CreateScript is used that we clear any previous server publisher list timestamp so that the client downloads the server pub list (since it was previously downloaded to old DB)

Reply:
👍